### DISCUSSION TITLE
How can I determine what is causing a style?

### DISCUSSION THREAD

**Discussion by Lastofthefirst** [lllyasviel/Fooocus#2730] [Q&A] [answered]
I like the lineart results I get in Fooocus for coloring pages. Heres a funny example made by mistake:

![Screenshot_20240407_140357](https://github.com/lllyasviel/Fooocus/assets/47003973/c0bfeec4-4a23-44b7-a8bf-b92309bed93d)

How can I determine what is being done to add the style so that i can draw on it in a comfyui workflow? Is it a lora? Is it adding to the prompt?

**Comment by mashb1t** [comment, marked as answer]
Styles add keywords to positive and negative prompt. You can find the SAI Line Art style here: https://github.com/lllyasviel/Fooocus/blob/main/sdxl_styles/sdxl_styles_sai.json#L53

**Reply by Lastofthefirst** [reply]
Great thanks! I was searching the repo for SAI-lineart, however its in there as sai-line art lol.
